### DISCUSSION TITLE
"Cannot convert"

### DISCUSSION THREAD

**Discussion by RampantPanda** [apprenticeharper/DeDRM_tools#1888] [Q&A] [answered]
Hi,
- calibre 5.11
- kubuntu 21.10
- DeDRM 7.21
- trying to dedrm a kindle ebook downloaded for my reader ("Download and transfer via USB")
- I have inserted the kindle serial number to the plugins settings under eInk Kindle ebooks

When trying to convert from AZW3 to MOBI (or any other format), I get the error:
# "Cannot convert <book name>
This book is locked by DRM. To learn more about DRM and why you cannot read or convert this book in calibre, click here."

I know I must be missing something obvious, but I have been able to do this previously with another installs, so I can't think of what I'm doing wrong here?

Thanks!

**Comment by RampantPanda** [comment, marked as answer]
OK, I installed the flatpak, which is version 5.29, and that works. So whatever the issue was, looks like it's fixed already. Thanks a bunch for a great plugin!

**Reply by ElleKayEm** [reply]
Yes, if you were using the calibre included with your Linux distro, that may have been the issue.
